### PR TITLE
support case where ssh private id_rsa key is locked with passphrase

### PR DESCRIPTION
**Pull request by jack-likes-to-code**

### CODE REVIEW

**Comment by jack-likes-to-code** [comment]
This code has some issues where a bad ssh passphrase + validate button causes there error to show up and then get redirected to the Day2 login page.  I have no idea why this is happening.

To require SSH passphrasing, back up your ~/.ssh/id_rsa and id_rsa.pub files.  Then use ssh-keygen to create a new one to replace those files, but make sure you enter a passphrase instead of just leaving it blank.

File: src/pages/ValidateConfigFiles.js [hunk 1]
```diff
@@ -220,35 +220,53 @@ class ValidateConfigFiles extends Component {
...
-      });
-  }
+    if (!this.props.sshPassphrase) {
+      this.props.enableNextButton(false);
+      this.setState({valid: INVALID, invalidMsg: translate('validate.config.sshPassphrase.missing')});
+      this.props.disableTab(false);
+      this.props.enableBackButton(true);
+    } else {
+      // set the password and validate
+      var password = {'password': this.props.sshPassphrase};
```
Comment (jack-likes-to-code) [inline src/pages/ValidateConfigFiles.js]: this should be 'let'

**Comment by jeremy-moffitt** [comment]
I'm confused about the state of this PR, is it ready for review or awaiting additional changes?

**Comment by jack-likes-to-code** [comment]
@jeremy-moffitt It's ready for code review, but if you want to test it, too, then you'll have to wait until the backend portion (https://gerrit.suse.provo.cloud/4279) is merged which Depends-On: a fix needed in CI (https://gerrit.suse.provo.cloud/4483)

**Comment by jack-likes-to-code** [comment]
Now that the backend changes to support this is in, this is ready for ready for review/approval

**Pull request merged**